source/ShipInfoDisplay.cpp

```diff
@@ -124,8 +125,8 @@ void ShipInfoDisplay::UpdateAttributes(const Ship &ship, const Depreciation &dep
 	
 	const Outfit &attributes = ship.Attributes();
 	
-	int64_t fullCost = ship.Cost();
-	int64_t depreciated = depreciation.Value(ship, day);
+	int64_t fullCost = depreciation.Value(ship, Depreciation::FullCostDay(), planet);
```

Comment:
Using depreciation to get the full cost of a ship is ... hacky. Might want to make it a function of `Ship` instead.

Reply:
I'm a hacker what do you want ? more seriously ok but it'll mean that function doing the same thing depreciation does pretty much
I agree this isn't a proper way to do it, will add a ship.LocalCost(planet) function